ext/storage_sources/gcp_store/README.md

```diff
@@ -0,0 +1,87 @@
+# WiredTiger's GCP Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored
+in Google Cloud Storage using WiredTiger’s provided internal abstraction for storing data in an
+object storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the GCP extension enabled.
+
+### Requirements
+* CMake 3.11 or higher
+* G++ 8.4 or higher
+* Abseil LTS 20230125
```

Comment:
I don't think the package requirements are correct for google. It seems we need for abseil we need 20200923, Patch 3, nlohmann_json at least 3.4.0, and crc32c at least 1.0.6.

Reply:
addressed in https://github.com/wiredtiger/wiredtiger/pull/8803/commits/9bb2ddf2eecf6aadd956b92142a8cb570df14b8c